templates/onboarding.html

```diff
@@ -0,0 +1,12 @@
+<p>
+    👋 Hello there! You have just joined a room bridged to Slack, which means messages you send
+    will be automatically bridged to Slack, and messages sent from Slack will appear in the room.
+    This room serves as your control center for configuring the bridge, and is accessible
+    only to you.
+</p>
+
+<p>
+    The bridge is not yet connecting direct messages, private channels or various other nice
+    things like reactions to your account. In order to do that, please type <code>login</code> in order
+    to connect your account to the bridge. You may also type <code>help</code> for other commands.
```

Comment:
Technically typing doesn't do the job. You gotta press Enter as well. 🤓
```suggestion
    things like reactions to your account. In order to do that, please send me the message <code>login</code>
    in order to connect your account to the bridge. You may also send <code>help</code> for other commands.
```